build/build.go

```diff
@@ -90,7 +90,10 @@ func Import(path string, mode build.ImportMode, installSuffix string, buildTags
 	return pkg, nil
 }
 
-func Parse(pkg *build.Package, fileSet *token.FileSet) ([]*ast.File, error) {
+// parse parses and returns all .go files for given pkg, including all transitive dependencies.
```

Comment:
> This is why describing what this func does will helpful, since it takes time and effort to figure it out from code.

I still can't believe I got it all wrong. I literally took 3-5 minutes carefully reading and looking over the func code to try to understand what exactly it does/is responsible for in order to summarize it, and came up with a completely wrong statement about dependencies. Sorry about that.